packages/components/src/components/input/input.scss

```diff
@@ -240,7 +240,9 @@
         max-width: calc(100% - 1.4rem);
     }
     &:not(&--no-placeholder) {
-        &__label {
+        $parent: &;
```

Comment:
Using parent variable to fix [this](https://github.com/sass/sass/issues/2228) breaking change.